### ISSUE TITLE
Bug: Theme switched in the Dark mode and light mode.

### ISSUE THREAD

**Issue by Divyanshgarg-04** [Sriparno08/Openpedia#229]
when the mode is in light mode it shows the dark theme and when the mode is in dark mode it shows the light theme

<img width="927" alt="image" src="https://github.com/Sriparno08/Openpedia/assets/120127517/2e705ea4-c514-4b93-9fb5-20fbe0d04f83">

**Comment by Divyanshgarg-04** [comment]
@Sriparno08 I would love to work on this issue under JWOC. Please assign this issue to me

**Comment by vishal2005025** [comment]
I have read the issue and I would like to work on this issue.
Please assign it to me under JWOC'2024.

**Comment by Sriparno08** [comment]
@Jenis07 You're assigned.

@Divyanshgarg-04 You're already assigned to another issue. Once your PR for it is merged, you will be considered for other issues.

**Comment by Divyanshgarg-04** [comment]
@Sriparno08  After my PR gets merged can you assign this issue to me.

**Comment by vermaanurag1532** [comment]
please Assign it to me
